src/vs/base/browser/ui/baseActionWidget/baseActionWidget.ts

```diff
@@ -0,0 +1,87 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Disposable, IDisposable, MutableDisposable } from 'vs/base/common/lifecycle';
+
+export interface ActionSet<T> extends IDisposable {
```

Comment:
Does this need to move into `base`? It seems to only be used at high levels, not in `base`

Reply:
I see what you mean - base action widget is not needed